### PR TITLE
Added action-security.yml for dlang-communtity/setup-dlang in KB

### PR DESCRIPTION
**Pull request by h0x0er**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,8 @@
 # vendor/
 testfiles/.DS_Store
 .DS_Store
-.vscode
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (varunsh-coder) [inline .gitignore:18]: @h0x0er can you please fix the .gitignore file?

Comment (h0x0er) [reply .gitignore]: @varunsh-coder I fixed the issue and also added a new action-security.yml

**Comment by varunsh-coder** [comment]
@h0x0er are you done with this? If so, I can merge.

**Comment by h0x0er** [comment]
@varunsh-coder yes I am done.

**Pull request merged**